capa/features/extractors/dnfile/helpers.py

```diff
@@ -370,6 +370,7 @@ def resolve_nested_typeref_name(
     # If the ResolutionScope decodes to a typeRef type then it is nested
     if isinstance(typeref.ResolutionScope.table, dnfile.mdtable.TypeRef):
         typeref_name = []
+        typeref_tb = []
```

Comment:
I'm not sure what `_tb` means, please add a comment or consider updating the name to something more obvious like `_visited`.